okio-kotlin/src/main/java/okio/Okio.kt

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2018 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:Suppress("NOTHING_TO_INLINE") // Aliases to public API.
+
+package okio
+
+import java.io.File
+import java.io.InputStream
+import java.io.OutputStream
+import java.net.Socket
+import java.nio.file.OpenOption
+import java.nio.file.Path
+
+/**
+ * Returns a [BufferedSource] from this [Source].
+ *
+ * @see Okio.buffer
+ */
+inline fun Source.buffer(): BufferedSource = Okio.buffer(this)
+
+/**
+ * Returns a [BufferedSink] from this [Sink].
+ *
+ * @see Okio.buffer
+ */
+inline fun Sink.buffer(): BufferedSink = Okio.buffer(this)
+
+/**
+ * Returns a [Sink] that writes to this [OutputStream].
+ *
+ * @see Okio.sink
+ */
+inline fun OutputStream.sink(): Sink = Okio.sink(this)
+
+/**
+ * Returns a [Sink] that writes to this [Socket].
+ *
+ * @see Okio.sink
+ */
+inline fun Socket.sink(): Sink = Okio.sink(this)
+
+/**
+ * Returns a [Source] that reads from this [InputStream].
+ *
+ * @see Okio.source
+ */
+inline fun InputStream.source(): Source = Okio.source(this)
+
+/**
+ * Returns a [Source] that reads from this [Socket].
+ *
+ * @see Okio.source
+ */
+inline fun Socket.source(): Source = Okio.source(this)
+
+/**
+ * Returns a [Sink] that writes to this [File].
+ *
+ * @see Okio.sink
+ */
+inline fun File.sink(): Sink = Okio.sink(this)
+
+/**
+ * Returns a [Sink] that appends to this [File].
+ *
+ * @see Okio.appendingSink
+ */
+inline fun File.appendingSink(): Sink = Okio.appendingSink(this)
```

Comment:
could also be `sink(append = true)` ?

Reply:
it’s less desirable in Java because there’s no named parameters

Reply:
Yaaaaaas

Reply:
Done